symphony/content/content.blueprintsdatasources.php

```diff
@@ -331,7 +331,7 @@ public function __form()
         $fieldset = new XMLElement('fieldset');
         $this->setContext($fieldset, array('sections', 'system'));
         $fieldset->appendChild(new XMLElement('legend', __('Error Conditions')));
-        $p = new XMLElement('p', __('Enabling these conditions will send a <code>404 page not found</code> error.'));
+        $p = new XMLElement('p', __('If enabled conditions are met, a <code>"404 page not found"</code> error will be sent.'));
         $p->setAttribute('class', 'help');
```

Comment:
(The following not meant to be understood as a criticism!  I like to understand and learn!)

Ok @michael-e I want to be picky ;)

1. Result is related to the causality.
2. Yeild feels more like a side-effect thing.

> It is a response.

Yup. Responses are SENT and RECEIVED.

Random example in the spec: https://tools.ietf.org/html/rfc7231

> The 100 (Continue) status code indicates that the initial part of a
   request has been received and has not yet been rejected by the
   server.  The server intends to send a final response after the
   request has been fully received and acted upon.

Boy am I having fun 👍